### PR TITLE
Fix: UI nits in multisig creation

### LINKED ISSUE #2531: Multisig. UI issues. Create ms flow
### 1. Create ms popup
Image

[Layouts reference](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=369-58126&t=bKjB2kBmhRjZrNaX-1)

1. align to one line and update text of description 
2. Close button state of is hover on initial screen enter
3. update placeholder text: "Choose wallet name" -> "Enter name"
4. update text
5. add Back button
6. skeleton place is shifted close to Continue
7. "Additional costs" -> "Additional cost"

### 2. Create ms on the network popup
Image

[Layouts reference](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=369-58363&t=bKjB2kBmhRjZrNaX-1)

1. NO NEED TO FIX (update popup title)
2. align and update description text
3. Label: Name -> Signatory name
4. Label: Your account -> Address
5. update placeholder text
6. plus button is wrong color
7. update placeholder text: Select threshold -> Select number
8. maybe disable threshold dropdown when <2 signatories inserted ???
9. update text
10. shifted skeleton place
11. Continue button is active anytime

### 3. Create ms editing fields
Image

[Layouts reference](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=1105-46243&t=bKjB2kBmhRjZrNaX-1)

1. when input address: inserted name is replaced with placeholder, name field is not editable
2. too big remove icon
3. shifted skeleton place

### 4. Confirm ms popup

Image

[Layouts reference](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=976-125596&t=bKjB2kBmhRjZrNaX-1)

1. close button is in hover state
2. no address under the name

### 5. Wallet details

- Singlechain multisig looks like this, which is not according to layout: network should be in the title with threshold info, there are two tabs: "Account and signatories" and "Delegated authorities"

Image

[Layouts reference](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=383-77731&t=GP7uXMP5hmNOy6f7-1)

- Multichain multisig looks like this, also not according the layout: all multisig wallets are separate in each network, so in network 	 only one network is possible

Image
Image

[Layouts reference](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=907-52209&t=V2B6hpWHF4LJPQL4-1)

Image
Image
Image

1. function button is in hover state
2. Remove treshold info, "Your accounts 1" -> "Account"
3. Contacts -> Signatories
4. Wallet name should be near Signatory address

### 6. Operations

Image

[Layouts reference](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=445-31031&t=nNUCyVpyhtqd8KiY-1)

1. wrong icon
2. missing dialog icon
3. missing Sender
4. Call Hash -> Call hash
5. Call Data -> Call data
6. missing i icon

### PR DESCRIPTION
**Pull request by Tbaut**
closes #2531

Create ms popup

   1 -  [text updated](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=815-41282&t=uAoUuDlNAYsyiGXd-4) 
    4 - [text updated](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=815-41282&t=uAoUuDlNAYsyiGXd-4) 
    5 - button added
    
image

Create ms on the network popup
     2- [text updated](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=815-41427&t=uAoUuDlNAYsyiGXd-4) and on 1 line.
     
Create ms editing fields
     1- fix for multiple own signatories

image

### CODE REVIEW

File: src/renderer/shared/i18n/locales/en.json
Comment (leohar) [inline src/renderer/shared/i18n/locales/en.json]: according to [layouts](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=815-41410&t=T99kW8RMsCtXdMqz-1) the string is: `Step 2. Assign the signatories and set the signatory threshold for your multisig.`

File: src/renderer/shared/i18n/locales/en.json
Comment (leohar) [inline src/renderer/shared/i18n/locales/en.json]: according to [layouts](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=815-41265&t=T99kW8RMsCtXdMqz-1) the string is: `Step 1. Select the network and the name of your multisig.`

Comment (Tbaut) [inline src/renderer/shared/i18n/locales/en.json]:
```suggestion
    "nameNetworkDescription": "Select the network and the name of your multisig",
```

File: src/renderer/shared/i18n/locales/en.json
Comment (Tbaut) [reply src/renderer/shared/i18n/locales/en.json]: Damn I was looking at the wrong. Thank you 
```suggestion
    "signatoryThresholdDescription": "Assign the signatories and set the signatory threshold for your multisig.",
```

**Review by tuul-wq — APPROVED** [review]

**Review by pgolovkin — APPROVED** [review]

**Comment by Tbaut** [comment]
I'll wait for @leohar to approve before merging, so that there's less work for both, and this merge can actually close the underlying issue.

**Review by Asmadek — APPROVED** [review]

File: src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx
Comment (sokolova-an) [inline src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx:73]: Do you think it is reasonable to use useMemo here? It doesn't look like an expensive operation and have a lot of dependencies

**Review by sokolova-an — APPROVED** [review]

File: src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx
Comment (Tbaut) [reply src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx:73]: I've been on the fence for long about these in the Dapps I built, but at the end of the day, `useMemo` and `useCallback` are always doing more good than bad from my experience, and I've come to use them absolutely everywhere. I know there are blog post out there saying that it's more costly to read, and isn't always needed. My stance is that it's easier to forget and have performance issues, than to use it everywhere and not have to worry about it.

**Review by leohar — APPROVED** [review]

**Pull request merged**
